### PR TITLE
feat: i18n

### PR DESCRIPTION
**Pull request by raymondmuller**
Saw this as a feature request so decided to jump in and add it.
Quite a big pull request as I've also added Spanish translations for most strings.
Hope it serves anyone!

i18n

### CODE REVIEW

**Comment by jean-moldovan** [comment]
@raymondmuller , please sync w/ master

**Comment by raymondmuller** [comment]
@jean-moldovan done

File: src/components/layout/sidebar/Sidebar.vue [hunk 1]
```diff
@@ -62,17 +59,10 @@
         }
       }
     },
-
-    watch: {
```
Comment (jean-moldovan) [inline src/components/layout/sidebar/Sidebar.vue]: Hm, we want menu to stay expanded after page refresh. Please refer to `http://vuestic.epicmax.co/#/ui/buttons` as an example.

File: src/components/tables/Table.vue [hunk 2]
```diff
@@ -80,16 +80,16 @@
...
           <div class="table-responsive">
             <table class="table table-striped table-sm color-icon-label-table">
               <thead>
               <tr>
                 <td></td>
-                <td>Name</td>
-                <td>Email</td>
-                <td>City</td>
-                <td align="right">Score</td>
+                 <td>{{'tables.headings.name' | translate}}</td>
```
Comment (jean-moldovan) [inline src/components/tables/Table.vue]: extra space

**Comment by raymondmuller** [comment]
@jean-moldovan Should be OK now. Synced with master as well.

**Pull request merged**